apps/ewallet_db/lib/ewallet_db/pre_auth_token.ex

```diff
@@ -85,21 +97,23 @@ defmodule EWalletDB.PreAuthToken do
   def generate(_, _, _), do: {:error, :invalid_parameter}
 
   @doc """
-  Retrieves an auth token using the specified token.
+  Retrieves a pre auth token using the specified token.
   Returns the associated user if authenticated, :token_expired if token exists but expired,
   or false otherwise.
   """
   def authenticate(token, owner_app) when is_atom(owner_app) do
     token
     |> get_by_token(owner_app)
-    |> return_user()
+    |> AuthExpirer.expire_or_refresh(get_lifetime())
```

Comment:
@T-Dnzt @sirn Not sure if it's a valid concern. This code would be doing 1 DB update on auth_token for each request.

Also the activity log will be spammed.

Maybe we should throttle down a bit?

Reply:
1 DB per request is fine, but I think we should do something with activity log spam.

Reply:
Security comes at a cost, so it's fine to have that DB request. Can disable activity log though, unless we're doing an expiration.